### ISSUE TITLE
What the detail about loss function in training process 

### ISSUE THREAD

**Issue by Xpangz** [yuhsuanyeh/BiFuse#14]
Hey! I find the paper doesn't show loss function in training stage completely, it just introduce reverse Huber loss for optimizing predictions from both Be and Bc, what about the next two stage?
